models/fixtures/user.yml

```diff
@@ -7,8 +7,7 @@
   full_name: User One
   email: user1@example.com
   email_notifications_preference: enabled
-  passwd_hash_algo: argon2
-  passwd: a3d5fcd92bae586c2e3dbe72daea7a0d27833a8d0227aa1704f4bbd775c1f3b03535b76dd93b0d4d8d22a519dca47df1547b # password
+  passwd: "$argon2$65536$2$8$50$a3d5fcd92bae586c2e3dbe72daea7a0d27833a8d0227aa1704f4bbd775c1f3b03535b76dd93b0d4d8d22a519dca47df1547b" # password
```

Comment:
Just store parameters on column `passwd_hash_algo` with a suffix like `argon2$65536$2$8$50` and if no parameter then default will be used so that it will be compatible with old version and no migrations needed.

Reply:
Sounds legit. Implemented it that way now.